src/main/java/com/onarandombox/MultiverseCore/api/operation/SimpleOperationResult.java

```diff
@@ -34,22 +34,22 @@ public static ActionResult forFailure(String name) {
      * @param isSuccessful  true if the result is successful.
      * @return The new ActionResult.
      */
-    public static ActionResult of(String name, boolean isSuccessful) {
-        return new SimpleActionResult(name, isSuccessful);
+    public static OperationResult of(String name, boolean isSuccessful) {
+        return new SimpleOperationResult(name, isSuccessful);
     }
 
     private final String name;
-    private final boolean isSuccessful;
+    private final boolean booleanState;
```

Comment:
I think `isSuccessful` is way more meaningful.

Reply:
ok ig i will change back then